user-service/src/main/java/rs/edu/raf/banka1/configuration/SpringSecurityConfig.java

```diff
@@ -41,7 +41,11 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .authorizeHttpRequests((authz) ->
                                 authz
 //                            .requestMatchers(HttpMethod.OPTIONS, "/**").permitAll()
-                                        .requestMatchers(antMatcher("/auth/**")).permitAll()
+                                        .requestMatchers(antMatcher("/auth/**"),
```

Comment:
ajde dodaj i za onu activate token putanju

Reply:
/user/activate/**

Reply:
![image](https://github.com/RAF-SI-2023/Banka-1-Backend/assets/75037128/4bf76c71-4f35-4e4a-a73f-e7603c96596d)

Can you just check with current version of the source if these permissions are present because when I did:
`git pull upstream sprint-2` I got these from the picture above.

Reply:
no, its not present, it is a bug, when user sets password it doesn't have jwt token